src/initializers/bull/bull-metrics.ts

```diff
@@ -0,0 +1,42 @@
+import { list, queuePath } from './bull-helper';
+import { getLogger } from '../log4js';
+import * as newrelic from 'newrelic';
```

Comment:
🔧  This might be required before the application has started. Some application might not have newrelic installed it's an optional peer-depencency. 
Use this instead https://github.com/Workable/orka/blob/338db674819e9bdc5bedf227a50669509e8fb9c6/src/initializers/newrelic/index.ts#L13

Reply:
Fixed it.Thank you!